django/thunderstore/community/models/community_site.py

```diff
@@ -98,6 +102,15 @@ def save(self, *args, **kwargs):
     def full_url(self) -> str:
         return f"{settings.PROTOCOL}{self.site.domain}/"
 
+    @cached_property
```

Comment:
Not sure if this is allowed to be a property, Django works with a callable at least. Good idea otherwise though.

Reply:
Works in ´manage.py shell´ 🤷